src/labels/CustomLabels.labels

```diff
@@ -4985,12 +4985,12 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <value>Batch Number Formats</value>
     </labels>
     <labels>
-        <fullName>batchNumberSettingsSave</fullName>
```

Comment:
Are you intending to remove the "batchNumberSettingsSave" label and replace it with the "batchNumberSettingsActivate" label?

Reply:
Yep, that was the intent - I just renamed it when I made this change end of last sprint.  Let me know if this will be ok or if you think it will cause any issues - I think this is the same way you have/had it in the other labels pr.